bindings/cs/rl.net/TraceLogEventArgs.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Rl.Net
+{
+    public class TraceLogEventArgs
+    {
+        public TraceLogEventArgs(int logLevel, string msg)
+        {
+            LogLevel = logLevel;
+            Message = msg;
+        }
+
+        public int LogLevel { get; }

```

Comment:
Use an enum instead of meaningless int.

Reply:
This is to align with rl c++ implementation. The flexibility of using enum for LogLevel should be provided on the caller side I think. See: https://msazure.visualstudio.com/One/_git/AIR-MSRAI-DecisionService/pullrequest/1389156?_a=overview and trace_logger.h

Reply:
Have the enum in rl.net now.